### ISSUE TITLE
[Bug?]: Behaviour on non-fullscreen, auto-hidden menubar

### ISSUE THREAD

**Issue by kappesante** [jordanbaird/Ice#286]
### Check Existing Issues

- [X] I have searched existing issues, and this issue is not a duplicate

### Description

I can see the scroll and the click triggers only working if the focus application is fullscreen. If not, i have to click on the ice menu item to show the hidden items.
At this point it may not be a bug, it may work as intended, but it was working until 0.10.2 maybe.

### Steps to Reproduce

1. having the focus application not in fullscreen
2. having the menubar auto hidden

### Ice Version

0.10.4

### macOS Version

14.5

### Screenshots

[https://www.loom.com/share/58bdb6fdec894daf88414cc10d656966?sid=d8283df9-164d-46aa-bf90-74d9d389890f](url)

**Comment by jordanbaird** [comment]
It works fine on my machine. Have you tried restarting both Ice and your computer?

**Comment by kappesante** [comment]
yep, and i did try many configurations, disabling other menu elements, uninstall and reinstall, all the usual stuff.
the automatism is gone since 0.10.2. 0.10.1 was working fine.

**Comment by jordanbaird** [comment]
Strange...I went back and looked at the changes between 0.10.1 and 0.10.2, and I don't see anything that would have affected that behavior. I'm going to say this is definitely a bug though.

**Comment by kappesante** [comment]
it's quite odd, yeah. also, if it helps, having a never-hidden menubar makes the auto show-hide working, but you have to restart ice.
i tried to re-enable the auto-hide menubar and ice keeps working until you restart it, or logout.
